mep/people/tests/test_models.py

```diff
@@ -58,6 +58,17 @@ def test_viaf(self):
         assert isinstance(pers.viaf, ViafEntity)
         assert pers.viaf.uri == pers.viaf_id
 
+    def test_short_name(self):
+        # should return up to comma for names with comma
+        pers = Person(sort_name='Casey, Jim')
+        assert pers.short_name == 'Casey'
+        # should just return the name if no comma
+        pers.sort_name = 'Jim Casey'
+        assert pers.short_name == 'Jim Casey'
+        # if no sort name should be None
+        pers.sort_name = None
+        assert pers.short_name is None
```

Comment:
Please also test on a name with a parenthetical and no comma (e.g. **H. D. (Hilda Doolittle)** or **A. E. (George William Russell)**) and a name with both (e.g. **Abrams, M. H. (Meyer Howard)**)